### PR TITLE
Add link and address to email footer

### PR DESCRIPTION
**Pull request by Baggerone**

### CODE REVIEW

File: application/templates/mail/_customer_footer.plush.html [hunk 1]
```diff
@@ -23,9 +23,15 @@
...
 	<div style="background-color: #ebeef2; padding: 16px;">
 		<div style="font-size: 9px; margin-bottom: 8px;">
-			<%= appName %> is a service of SIL International.
+			<span style="margin-right: 10px;">
+				<a href="<%= uiURL %>/home">Your Account</a>
+			</span>
 		</div>
 		<div style="font-size: 9px; margin-bottom: 8px;">
+			<%= appName %> is a service of SIL International.<br>
+			SIL International, 7500 W Camp Wisdom Rd, Dallas, TX 75236, USA
```
Comment (Schparky) [inline application/templates/mail/_customer_footer.plush.html:32]: I think it's strange that the mockups have this address, when the sil.org site doesn't even give this information. If we really want it here, maybe put it in an environment variable.

Comment (Baggerone) [reply application/templates/mail/_customer_footer.plush.html:32]: The sil.org site does give the information on the `contact` page.

If we don't limit the org name to an environment variable, I doubt it's worth adding one for the address.

Comment (Schparky) [reply application/templates/mail/_customer_footer.plush.html:32]: Oh, sorry. Not sure how I missed that.

**Review by Schparky — APPROVED** [review]

File: application/templates/mail/_customer_footer.plush.html [hunk 2]
```diff
@@ -23,9 +23,15 @@
 	</div>
 	<div style="background-color: #ebeef2; padding: 16px;">
 		<div style="font-size: 9px; margin-bottom: 8px;">
```
Comment (jason-jackson) [inline application/templates/mail/_customer_footer.plush.html:25]: If all of these internal divs have `font-size: 9px;`, why not put it in parent div style? (or is it an email thing?)

**Review by jason-jackson — APPROVED** [review]

**Pull request merged**